bench/Sender/Main.hs

```diff
@@ -56,7 +57,7 @@ main = do
     loadLogConfig logsPrefix logConfig
     setLocaleEncoding utf8
 
-    Right transport_ <- TCP.createTransport "0.0.0.0" "3432" TCP.defaultTCPParameters
+    Right transport_ <- TCP.createTransport "0.0.0.0" "127.0.0.1" "3432" TCP.defaultTCPParameters
```

Comment:
@avieth binding on oublic ip night not work on some systems. E.g. if you don;t have interface with this ip on your system, like we encountered on AWS